### ISSUE TITLE
Non-pretrained results

### ISSUE THREAD

**Issue by Hong753** [linhaowei1/TPLR#3]
Hello, I'm trying to reproduce your results without using a pre-trained network (i.e., using ResNet-18).
I've followed https://github.com/k-gyuhak/WPTP implementation to use HAT + TPL's adapter, but I'm unable to reproduce the results reported on your paper (Table 8, Appendix).
Could you upload the training code for such results?

Thanks

**Comment by linhaowei1** [comment]
Certainly. It might take some time to clean and refactor the code. Please be patient.
